### PR TITLE
Add more system checks for url like settings

### PR DESCRIPTION
**Pull request by rwrzesien**
Resolves https://github.com/golemfactory/concent/issues/589

### CODE REVIEW

File: concent_api/concent_api/tests/test_storage_cluster_address_check.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+from django.test                import override_settings
+from django.conf                import settings
+from django.test                import TestCase
+
+from concent_api.system_check   import create_error_36_storage_cluster_address_does_not_end_with_slash
+from concent_api.system_check   import create_error_38_storage_cluster_address_is_not_valid_url
+from concent_api.system_check   import create_error_39_storage_server_internal_address_is_not_set
+from concent_api.system_check   import check_settings_storage_cluster_address
```
Comment (pawelkisielewicz) [inline concent_api/concent_api/tests/test_storage_cluster_address_check.py]: according to our coding style it should be without additional space, just:
```
from django.test import override_settings
from django.conf import settings
from django.test import TestCase

from concent_api.system_check import create_error_36_storage_cluster_address_does_not_end_with_slash
from concent_api.system_check import create_error_38_storage_cluster_address_is_not_valid_url
...
```

Comment (rwrzesien) [reply concent_api/concent_api/tests/test_storage_cluster_address_check.py]: :+1: done

File: concent_api/concent_api/system_check.py [hunk 2]
```diff
@@ -277,39 +309,35 @@ def check_settings_concent_features(app_configs, **kwargs):  # pylint: disable=u
 
 
 @register()
-def check_settings_storage_cluster_address(app_configs, **kwargs):  # pylint: disable=unused-argument
+def check_settings_storage_cluster_address(app_configs=None, **kwargs):  # pylint: disable=unused-argument
```
[the remark this answers is not in the document]
Comment (rwrzesien) [reply concent_api/concent_api/system_check.py]: Yes and yes, it has to met check interface.

File: concent_api/concent_api/system_check.py [hunk 3]
```diff
@@ -240,6 +240,38 @@ def create_error_35_additional_verification_time_multiplier_has_wrong_type(addit
     )
 
 
+def create_error_36_storage_cluster_address_does_not_end_with_slash():
+    return Error(
+        "STORAGE_CLUSTER_ADDRESS must end with '\''",
```
[the remark this answers is not in the document]
Comment (rwrzesien) [reply concent_api/concent_api/system_check.py]: :+1: done

File: concent_api/concent_api/system_check.py [hunk 4]
```diff
@@ -240,6 +240,38 @@ def create_error_35_additional_verification_time_multiplier_has_wrong_type(addit
...
+        hint=f"STORAGE_CLUSTER_ADDRESS must end with '\''.",
+        id="concent.E036",
+    )
+
+
+def create_error_37_storage_server_internal_address_does_not_end_with_slash():
+    return Error(
+        "STORAGE_SERVER_INTERNAL_ADDRESS must end with '\''",
```
[the remark this answers is not in the document]
Comment (rwrzesien) [reply concent_api/concent_api/system_check.py]: :+1: done

**Pull request merged**